README.md

```diff
@@ -261,7 +261,7 @@ Here is an example of the data that will be forwarded to Splunk in each instance
 
 ## Tasks
 
-Two tasks are provided for submitting data from a Bolt plan to Splunk. For clarity, we recommend using a different HEC token to distinguish between events from Puppet runs and those generated by Bolt. The Puppet Report Viewer app includes a `puppet:bolt` sourcetype to faciltate this. Currently SSL validation for Bolt communications to Splunk is not supported.
+Two tasks are provided for submitting data from a [Bolt plan](https://puppet.com/docs/bolt/latest/bolt_running_plans.html) to Splunk. For clarity, we recommend using a different HEC token to distinguish between events from Puppet runs and those generated by Bolt. The Puppet Report Viewer app includes a `puppet:bolt` sourcetype to faciltate this. Currently SSL validation for Bolt communications to Splunk is not supported.
```

Comment:
I would link to the higher level description of what a plan is and let users navigate their own way to running or writing or wherever they want to go. https://puppet.com/docs/bolt/latest/plans.html